IFC4x3/PropertyConstants/h/HEXAGONAL_2Jr9$rap1CU8tgt$57FRl1.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HEXAGONAL_2Jr9$rap1CU8tgt$57FRl1" Name="HEXAGONAL" UniqueId="93d49ff5-9330-4c78-8dea-dff1473dbbc1" />
```

Comment:
Description.

Reply:
Fixed in [26b41fa](https://github.com/bSI-InfraRoom/IFC-Specification/pull/828/commits/26b41fa080bec029809d1e01e57a3b4e2c4d9122)